dask/dataframe/core.py

```diff
@@ -526,7 +527,7 @@ def drop_duplicates(
             split_out_setup=split_out_setup,
             split_out_setup_kwargs=split_out_setup_kwargs,
             ignore_index=ignore_index,
-            **kwargs
+            **kwargs,
```

Comment:
These trailing commas are getting added by my pre-commit hook apparently because black now believes that this file is Python >= 3.6 only https://black.readthedocs.io/en/stable/the_black_code_style.html#trailing-commas